src/code/utilities/scales.ts

```diff
@@ -0,0 +1,2 @@
+export const fToC = (f: number) => (f - 32) * (5/9);
+export const cToF = (c: number) => (c * (5/9)) + 32;
```

Comment:
Check me on this, I think that's 9/5'ths for C -> F since C degrees are bigger than F degrees.